packages/server/tests/acceptance/ws/subscribe.spec.ts

```diff
@@ -232,6 +232,11 @@ describe('@web-socket Acceptance Tests', async function() {
 
             await new Promise(resolve => setTimeout(resolve, 1000));
         });
+
+        it('Connection TTL is enforced, should close all connections', async function() {
+            await new Promise(resolve => setTimeout(resolve, parseInt(process.env.WS_MAX_CONNECTION_TTL) + 1000));
```

Comment:
Doesn't this test require that a few connections be set up first to ensure they are closed off?

Reply:
Yes, at least 1 connection, one is opened on the beforeEach so its available on all tests, but I have added 2 more, asserted they are opened, wait, and they assert that there are none for more readability of the test itself.